scripts/ci/prep-local-docker-registry-credentials

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
 set -euvo pipefail
+
+# Creates Basic Authentication Context
+pushd tests/functional/artifacts/registry/
```

Comment:
if we're not gonna `popd` why not just use `cd`?

Reply:
purely habit :) does no harm though

Reply:
sure, i guess it's a nit picky comment haha